.github/workflows/iam-handle.yml

```diff
@@ -0,0 +1,58 @@
+name: 'aod-iam-handle'
+on:
+  workflow_call:
+    inputs:
+      workload_identity_provider:
+        description: 'The full identifier of the Workload Identity Provider, including the project number, pool name, and provider name.'
+        type: 'string'
+        required: true
+      service_account:
+        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials.'
+        type: 'string'
+        required: true
+      path:
+        description: 'The path to a YAML file that contains the IAM request.'
+        type: 'string'
+        required: true
+      start_time:
+        description: 'The start time of the IAM permission lifecycle.'
+        type: 'string'
+        required: true
+      duration:
+        description: 'The IAM permission lifecycle, as a duration.'
+        type: 'string'
+        required: true
```

Comment:
Shouldn't we detect them automatically? We don't want workflow users to do that.

Reply:
Was thinking the other way around, make sense, will update.